Generate_Client/generate.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -eu
+
+origin="$(pwd)"
+cd ~
+if [ ! -d "./openapi-generator" ]
+  then
+        git clone https://github.com/openapitools/openapi-generator
+        cd ~/openapi-generator/
+        mvn clean package
+        exit 0
+else
+        cd ~/openapi-generator
+fi
```

Comment:
This whole block should be tweaked such that:
* If the openapi-generator is not available, the script fails early with a suitable error message
* It should not require that an openapi-generator dir exist in the user's home dir
* It should not "install" software without warning.

Because the system requires running a jar file, I suggest using a command line switch or environment variable that the user can provide to the script for the location of the openapi-generator. If the generator can't be found it could print out the instructions for cloning the repo and building the jar files.